homeassistant/components/modbus/climate.py

```diff
@@ -112,7 +112,16 @@ def __init__(
             CONF_TARGET_TEMP_WRITE_REGISTERS
         ]
         self._unit = config[CONF_TEMPERATURE_UNIT]
-
+        self._hvacmodeToIndexArray = {
```

Comment:
This looks like a constant, so why have an instance in each object....at the very least it should be a class variable.